app/components/PartialChallengeDetailsView.js

```diff
@@ -91,15 +66,49 @@ PartialChallengeDetailsView.propTypes = {
 }
 
 const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor:"lightgrey", 
-    alignItems:"center", 
-    justifyContent:"center"
+  startButton: {
+    backgroundColor: 'red',
+    height: 36,
+    width: 80,
+    margin: 6,
+    position: 'absolute',
+    right: 6,
+    borderRadius: 3
+  },
+  challengeDetails: {
+    flex: .2,
```

Comment:
One thought about this, If we care about users with tablets, this will potentially stretch this thing to be very tall. 20% of the height of a tablet is a lot different then 20% of a phone screen.